docs/faq.md

```diff
@@ -169,6 +169,13 @@ help but are outside of the scope of video.js.
 
 For content that must be highly secure [videojs-contrib-eme][eme] adds DRM support.
 
+## Q: Can I turn off video.js logging?
```

Comment:
Added debugging guide, which includes that information. And added a link to that guide here.